waypoint.hcl

```diff
@@ -425,11 +425,11 @@ app "reward-api" {
       service_port        = 8000
       region              = "us-east-1"
       memory              = "512"
-      cluster             = "reward-api-staging"
+      cluster             = "reward-api"
       count               = 2
       subnets             = ["subnet-004c18e7177f0a9a2", "subnet-053fc89a829849140"]
-      task_role_name      = "reward-api-staging-ecr-task"
-      execution_role_name = "reward-api-staging-ecr-task-executor-role"
+      task_role_name      = "reward-api-ecs-task"
+      execution_role_name = "reward-api-ecs-task-execution"
```

Comment:
Note. Since move to `aws-ecs` module and exclusion of `ecr-task-suport` module. Have pointed to different task names. See here https://github.com/cardstack/infra/pull/177

Reply:
@pcjun97